strongbox-rest-client/src/main/java/org/carlspring/strongbox/client/RestClient.java

```diff
@@ -392,7 +392,7 @@ public int addRepository(RepositoryForm repository,
      * @return
      * @throws java.io.IOException
      */
-    public ImmutableRepository getRepository(String storageId,
+    public Repository getRepository(String storageId,
```

Comment:
I see now. In this case, perhaps we should rename the `*Data` classes to `*Dto`...?

Reply:
@carlspring please have a look [this](https://github.com/strongbox/strongbox/pull/1359#discussion_r299744012) comment